### ISSUE TITLE
Move to Alpine 3.20 and a newer version of s6-overlay

### ISSUE THREAD

**Issue by javydekoning** [crazy-max/docker-rtorrent-rutorrent#353]
### Description

3.20 is out and the s6 overlay you are using is a version from 2021. 

https://github.com/just-containers/s6-overlay/releases?page=1

**Comment by crazy-max** [comment]
Do you have any issue with current s6 overlay v2? I didn't make the jump to s6 v3 yet as it needs a huge refactor but I added support for Alpine 3.20 in https://github.com/crazy-max/docker-alpine-s6/releases/tag/2.2.0.3-r24

**Comment by javydekoning** [comment]
No, no issues with s6 just stood out that it's an older version. Thanks!
